components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.v1/src/main/java/org/wso2/carbon/identity/api/server/configs/v1/core/ServerConfigManagementService.java

```diff
@@ -184,6 +184,20 @@ public ServerConfig getConfigs() {
             rememberMePeriod = rememberMeProp.getValue();
         }
 
+        String signingKeyAlias = null;
```

Comment:
can we create a private method
private String derriveSigningKeyAlias() or something